File: cloud-collect-core/src/main/java/com/windea/demo/cloudcollect/core/exception/InvalidUserException.java
package com.windea.demo.cloudcollect.core.exception;

/**
 * 非法的用户的异常。
 */
public class InvalidUserException extends RuntimeException {
	private static final long serialVersionUID = 867810939053316512L;

	private static final String message = "403 非法的用户！";

	public InvalidUserException() {
		super(message);
	}

	public InvalidUserException(Throwable throwable) {
		super(message, throwable);
	}
}

File: cloud-collect-core/src/main/kotlin/com/windea/demo/cloudcollect/core/exception/InvalidUserException.kt
package com.windea.demo.cloudcollect.core.exception

import com.windea.demo.cloudcollect.core.domain.enums.*

/**非法的用户的异常。 */
class InvalidUserException : RuntimeException {
	constructor() : super(ResponseResult.INVALID_USER.message)
	
	constructor(throwable: Throwable) : super(ResponseResult.INVALID_USER.message, throwable)
}
